airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -477,6 +481,32 @@ public int getNumberOfFileBuffers(final JsonNode config) {
   }
 
   public static void main(final String[] args) throws Exception {
+    if ("--write".equals(args[0])) {
+      final List<String> targetStrings = List.of("foo", "bar", "baz");
+      final BigQueryException e = new BigQueryException(42, "Not found: Table bar.baz was not found in location US at [53:5]");
+
+      final String mangledMessage =
+          "airbyte_destination_bigquery_error: "
+          + targetStrings.stream().reduce(
+              e.getMessage().replaceAll("\\d", "?"),
```

Comment:
this should probably actually be `\d+` to collapse multi-digit numbers into a single placeholder